### PR TITLE
Feature/custom options

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Now we can test with random value and random path!

### CODE REVIEW

File: worker_node/job/send.js [hunk 1]
```diff
@@ -164,11 +166,18 @@ class Send extends Base {
...
 
+      const tx = JSON.parse(JSON.stringify(baseTx));
       sendTxPromiseList.push(
           new Promise((resolve, reject) => {
             setTimeout((timestamp) => {
-              baseTx.timestamp = timestamp;
-              this.#ain.sendTransaction(baseTx).then(result => {
+              tx.timestamp = timestamp;
+              if (randomPath) {
+                tx.operation.ref = `${tx.operation.ref}/${i}`;
```
Comment (liayoo) [inline worker_node/job/send.js:175]: It seems the paths are not really 'random' but rather 'consecutive'. Could you consider renaming it to consecutivePath OR using something like Math.random() or uuid?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: worker_node/job/send.js (hunk 1 above)
Comment (cshcomcom) [reply worker_node/job/send.js:175]: Looks good! I'll change it!

**Comment by cshcomcom** [comment]
Thanks for the review! Now merging

**Pull request merged**